Main.java

```diff
@@ -0,0 +1,15 @@
+package com.company;
```

Comment:
A Basic comment on the overall commit message, Usually commit message must be related to the changes you made to the project, here you added solution for a question to the repository, here is my suggestion.
"Add files via upload" -> "Added solution to print a triangle"